### PR TITLE
Updating CAD and BOM to reflect slip clutch additions

### PR DESCRIPTION
**Pull request by krskim**
Added modified reel drive clamp, modified shaft adapter, modified reel hub adapter. Added slip clutch pulley CAD, slip clutch CAD. Modified the BOM to reflect new connectors and slip clutch parts.

### CODE REVIEW

**Comment by braingram** [comment]
Thanks for updating the CAD. I left some more comments on the updated instructions. The BOM updates look good as well. Would you update the BOM to reflect the new date and to include the stainless steel takeup rest mentioned in the instructions?

To help document these changes, would you also add a description of the changes included in this PR to the repo? A simple version of this might be to add a `CHANGES.md` file in the base of the repo and write a description of the changes in this PR (perhaps under a heading for `Dec 2023 - PR #3` or something along those lines.

File: CHANGES.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+Dec 2023 - PR #3
```
Comment (braingram) [inline CHANGES.md]:
```suggestion
# Dec 2023 - [Updating CAD and BOM to reflect slip clutch changes](https://github.com/htem/GridTapeStage/pull/3)
```

File: CHANGES.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+Hardware
```
Comment (braingram) [inline CHANGES.md]:
```suggestion
## Hardware
```

File: CHANGES.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+- Parts
+	- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
+	- Added slip clutch and slip clutch pulley CAD 
+	- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
+- Drawings
+	- Added drawings of updated CAD
+
+Docs
```
Comment (braingram) [inline CHANGES.md]:
```suggestion
## Docs
```

File: CHANGES.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+Dec 2023 - PR #3
+Hardware
+- Parts
```
Comment (braingram) [inline CHANGES.md]:
```suggestion
### Parts
```

File: CHANGES.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+	- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
+	- Added slip clutch and slip clutch pulley CAD 
+	- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
+- Drawings
+	- Added drawings of updated CAD
+
+Docs
+- Updated BOM:
```
Comment (braingram) [inline CHANGES.md]:
```suggestion
### BOM
```

File: CHANGES.md [hunk 6]
```diff
@@ -0,0 +1,15 @@
...
+	- Updated CAD components
+	- Changed the material of the takeup_rest from aluminum to stainless steel
+	- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck
+- Updated Gridtape Assembly Instructions
```
Comment (braingram) [inline CHANGES.md]:
```suggestion
### Assembly Instructions

details about the assembly instructions changes...
```

File: CHANGES.md [hunk 7]
```diff
@@ -0,0 +1,15 @@
+Dec 2023 - PR #3
+Hardware
+- Parts
+	- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
+	- Added slip clutch and slip clutch pulley CAD 
+	- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
+- Drawings
```
Comment (braingram) [inline CHANGES.md]:
```suggestion
### Drawings
```

File: CHANGES.md [hunk 8]
```diff
@@ -0,0 +1,15 @@
+Dec 2023 - PR #3
+Hardware
+- Parts
+	- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
+	- Added slip clutch and slip clutch pulley CAD 
+	- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
```
Comment (braingram) [inline CHANGES.md]:
```suggestion

- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
- Added slip clutch and slip clutch pulley CAD 
- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
- Added missing counterbore to back of `reel_hub`

```

Is `reel_hub` the right part?

File: CHANGES.md [hunk 9]
```diff
@@ -0,0 +1,15 @@
...
+- Drawings
+	- Added drawings of updated CAD
+
+Docs
+- Updated BOM:
+	- Updated CAD components
+	- Changed the material of the takeup_rest from aluminum to stainless steel
+	- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck
```
Comment (braingram) [inline CHANGES.md]:
```suggestion

- Updated CAD components
- Changed the material of the takeup_rest from aluminum to stainless steel
- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck

```
Would you expand `Updated CAD components` above?

**Review by braingram — COMMENTED** [review]
Thanks for adding the changelog. I made a few suggestions to use more of the [syntax understood by github's markdown renderer](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax).

File: CHANGES.md (hunk 8 above)
Comment (krskim) [reply CHANGES.md]: Nope, it's reel_adapter.

```suggestion
- Updated the reel_drive_clamp and shaft_adapter CAD to their slip clutch varients
- Added slip clutch and slip clutch pulley CAD 
- Updated overall CAD assembly to include slip clutch and slip clutch adjacent CAD
- Added missing counterbore to back of `reel_adapter'
```

File: CHANGES.md (hunk 9 above)
Comment (krskim) [reply CHANGES.md]: Yeah! 
```suggestion
- Changed the reel drive plate and slip clutch shaft CAD file reference to reel_drive_clamp_slipclutch and shaft_adatper_slipclutch
- Removed components for old slipclutch
- Added current slipclutch
- Changed the material of the takeup_rest from aluminum to stainless steel
- Included 4-pin connectors for the load cell amplifier and the LED, and a 7pos housing receptacle for the LED buckpuck
```

**Pull request merged**